src/components/ExecuteButton.js

```diff
@@ -0,0 +1,183 @@
+            </Text>
+            <Text component={ TextVariants.small }>
+                 Connect your systems to Satellite to automatically remediate.
+            </Text>
+            <Button
+                key="download"
+                // eslint-disable-next-line no-console
+                variant='link' onClick={ () => console.log('TODO: add link') }>
+               Learn how to connect
+            </Button>
+        </TextContent>),
+    no_source: (<TextContent>
+        <Text component={ TextVariants.p }>
+            Cannot remediate - Satellite not configured
+        </Text>
+        <Text component={ TextVariants.small }>
+           Satellite not registered for Playbook execution
+        </Text>
+        <Button
+            key="register"
+            // eslint-disable-next-line no-console
+            variant='link' onClick={ () => console.log('TODO: add link') }>
+            Learn how to register Satelite
+        </Button>
+    </TextContent>),
+    no_receptor: (<TextContent>
+        <Text component={ TextVariants.p }>
+            <ExclamationCircleIcon
+                className="ins-c-remediations-connection-status-error ins-c-remediations-connection-status"
+                aria-label="connection status" />
+            Cannot remediate - Receptor not configured
+        </Text>
+        <Text component={ TextVariants.small }>
+            Configure Receptor to automatically remediate
+        </Text>
+        <Button
+            key="configure"
+            // eslint-disable-next-line no-console
+            variant='link' onClick={ () => console.log('TODO: add link') }>
+            Learn how to configure
+        </Button>
+    </TextContent>)
+})[status];
+
+const ExecuteButton = ({ isLoading, data, getConnectionStatus, remediationId, issueCount }) => {
+    const [ open, setOpen ] = useState(false);
+    const [ isUserEntitled, setIsUserEntitled ] = useState(false);
+    useEffect(() => {
+        window.insights.chrome.auth.getUser().then(user => setIsUserEntitled(user.entitlements.smart_management.is_entitled));
+    }, []);
+
+    const [ connected, disconnected ] = data.reduce(
+        ([ pass, fail ], e) => (e.connection_status === 'connected' ? [ [ ...pass, e ], fail ] : [ pass, [ ...fail, e ] ])
+        , [ [], [] ]
+    );
+
+    const rows = [ ...connected, ...disconnected ].map(con =>
+        ({ cells: [ con.executor_name || 'Direct connection', con.system_count, { title: styledConnectionStatus(con.connection_status) }]})
+    );
+    const connectedCount = connected.reduce((acc, e) => e.system_count + acc, 0);
+    const systemCount = data.reduce((acc, e) => e.system_count + acc, 0);
+
+    const pluralize = (number, str) => number > 1 ? `${number} ${str}s` : `${number} ${str}`;
+    return (isUserEntitled
+        ?  <React.Fragment>
+            <Button
+                onClick={ () => { setOpen(true); getConnectionStatus(remediationId); } }>
+        Execute playbook
```

Comment:
`playbook` should be capitalized

Reply:
Are you sure? I know that `Download Playbook` is using upper case P, but mock are consistently using lower case p.

Reply:
Yes, in the past the Ansible team asked us to always use capitalized "Playbook" everywhere.

Reply:
Our capitalization guide had been recommending lowercase, but I've updated based on this info that it should be uppercase. 
Mocks may be inconsistent while I work on getting them updated